fuzz/type_info_adt_set.c

```diff
@@ -57,8 +60,6 @@ alloc_scen(struct theft *t, void *type_env, void **output)
 		}
 	}
 
-	res->count = count;
```

Comment:
I wait to set the count until after because (conceptually) until they're ready, the count is zero. In some other generators (such as the NFA one) I have an early exit that drops the overall count, because it gives a better foothold for shrinking.

Reply:
Yes, I saw that.